modules/cms/twig/FrameworkNode.php

```diff
@@ -35,25 +35,35 @@ public function compile(TwigCompiler $compiler)
             $compiler
                 ->write("if (\$_minify) {" . PHP_EOL)
                 ->indent()
-                    ->write("echo '<script src=\"'. Request::getBasePath()
-                    .'/modules/system/assets/js/framework.combined-min.js\"></script>'.PHP_EOL;" . PHP_EOL)
+                    ->write("echo '<script async src=\"'. Request::getBasePath()
+                    .'/modules/system/assets/js/framework.combined-min.js\" importance=\"low\"></script>'.PHP_EOL;" . PHP_EOL)
+                    ->write("echo '<link rel=\"preload\" href=\"'. Request::getBasePath()
```

Comment:
Why were these added? Many instances of October load the framework assets at the end of the document where <link> would be invalid. Also, how does it have any effect at all appearing immediately after the asset is requested in the first place?

Reply:
I've tested this and the browser does preload them, have you tested it?